keyboards/system76/launch_1/rules.mk

```
@@ -0,0 +1,39 @@
+# MCU name
+MCU = atmega32u4
+
+# Allow flashing with usbasp
+AVRDUDE_MCU = $(MCU)
+
+# CPU frequency divided by two since AVR is at 3.3 V
+F_CPU = 8000000
+
+# External oscillator is 16 MHz
+F_USB = 16000000
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build options
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```